src/python/pants/auth/cookies.py

```diff
@@ -18,8 +20,9 @@ class Cookies(Subsystem):
   @classmethod
   def register_options(cls, register):
     super(Cookies, cls).register_options(register)
-    register('--path', advanced=True, fingerprint=True, default='~/.pants.cookies',
-             help='Path to file that stores persistent cookies.')
+    register('--path', advanced=True, fingerprint=True, default=None,
+             help='Path to file that stores persistent cookies. '
+                  'Defaults to <pants bootstrap dir>/auth/cookies.')
```

Comment:
Any reason not to just populate default above? `default=os.path.join(register.bootstrap.pants_bootstrapdir, 'auth', 'cookies')` should do it.

Reply:
Only that I didn't remember how to access pants_bootstrapdir... Done.